build/Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM registry.access.redhat.com/ubi7/ubi-minimal:7.6
+FROM registry-proxy.engineering.redhat.com/rh-osbs/ubi8-minimal:8.1-released
```

Comment:
It _may_ be better to use `8-released` instead of `8.1-released`? The former will choose the latest version of ubi8, while the latter will stick to the 8.1 stream.

Reply:
```suggestion
FROM registry-proxy.engineering.redhat.com/rh-osbs/ubi8-minimal:8-released
```